op-chain-ops/deployer/deployer.go

```diff
@@ -40,11 +40,19 @@ type Deployment struct {
 
 type Deployer func(*backends.SimulatedBackend, *bind.TransactOpts, Constructor) (*types.Transaction, error)
 
+// NewBackend returns a SimulatedBackend suitable for L1. It has
+// the latest L1 hardforks enabled.
 func NewBackend() *backends.SimulatedBackend {
```

Comment:
nit: Should we rename this to `NewL1Backend` to avoid people potentially using it for L2 by accident?

Reply:
Good call, addressed this